tests/cpp/test_learner.cc

```diff
@@ -143,7 +143,7 @@ TEST(Learner, Json_ModelIO) {
     for (int32_t iter = 0; iter < kIters; ++iter) {
       learner->UpdateOneIter(iter, p_dmat.get());
     }
-    learner->SetAttr("bset_score", "15.2");
+    learner->SetAttr("best_score", "15.2");
```

Comment:
```suggestion
    learner->SetAttr("base_score", "15.2");
```

Reply:
@hcho3 No, it's actually best score or best iteration.

Reply:
@hcho3 Used by binding to set this training information.  The binary IO is misusing it for storing other parameters.

Reply:
@hcho3 See : https://github.com/dmlc/xgboost/blob/208ab3b1ff22ac2d5d767961468016a17240268c/python-package/xgboost/callback.py#L219

Reply:
Got it.